api/service.go

```diff
@@ -0,0 +1,391 @@
+package api
+
+import (
+	"bytes"
+	"errors"
+	"net/http"
+	"strconv"
+	"strings"
+
+	"github.com/decred/dcrd/chaincfg/v3"
+	chainjson "github.com/decred/dcrd/rpc/jsonrpc/types/v3"
+	apiTypes "github.com/decred/dcrdata/v7/api/types"
+)
+
+type (
+	// Backend defines the backend type.
+	Backend string
```

Comment:
`Backend` is a generic term. Saying `Backend` defines the backend type, doesn't help. Looking at the type name and comment, I still have no idea what sort of data this type is meant to store.

Reply:
[Here's an example](https://github.com/decred/dcrwallet/blob/3b3e9e04b3c27123f8cb5bc3204924bbd0021e6c/wallet/network.go#L40-L46) of a custom type that is a backend, with sufficient commentary on what sort of backend it is.

```
// NetworkBackend provides wallets with Decred network functionality.  Some
// wallet operations require the wallet to be associated with a network backend
// to complete.
//
// NetworkBackend expands on the Peer interface to provide additional
// functionality for rescanning and filtering.
type NetworkBackend interface {
```